src/styles/blocks/stars.css

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(../../images/star.svg);
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
```

Comment:
Don't fix container size (if there is no such a requirement). However, in this case, since you're dealing with a background image inside a fixed-size element (the star), it's acceptable to set explicit widths and heights for the '.stars__star' class to ensure that the star images are displayed correctly.